README.md

```diff
@@ -148,6 +148,17 @@ by manually calling the appropriate vcvars file before running the bootstrap.
 > python x.py build
 ```
 
+### Building rustc with older host toolchains
+between the below versions of the host toolchains the rustc build fails unless
```

Comment:
```suggestion
It is still possible to build Rust with the older toolchain versions listed below, but only if the
```

Reply:
used between due to the upper version do not need the option to be set but was not that happy with that wording
shall I try to find the upper version the is not working and add them in the list below?